core/src/main/java/com/kin/ecosystem/core/Configuration.java

```diff
@@ -57,20 +63,29 @@ public static ApiClient getDefaultApiClient() {
 					@Override
 					public Response intercept(Chain chain) throws IOException {
 						Request originalRequest = chain.request();
-
 						final String path = originalRequest.url().encodedPath();
-						AuthToken authToken = null;
-						if (!path.equals(USERS_PATH)) {
-							authToken = AuthRepository.getInstance().getAuthTokenSync();
-						}
 
-						if (authToken != null) {
-							Request authorisedRequest = originalRequest.newBuilder()
-								.header(AUTHORIZATION, BEARER + authToken.getToken())
-								.build();
-							return chain.proceed(authorisedRequest);
-						} else {
+						if (path.equals(USERS_PATH)) {
 							return chain.proceed(originalRequest);
+						} else {
+							AuthToken authToken = AuthRepository.getInstance().getAuthTokenSync();
+							if (authToken != null) {
```

Comment:
make sure to request offer in Kin after register for the first time to pre fetch offers